src/bin/tikv-ctl.rs

```diff
@@ -781,6 +820,16 @@ impl DebugExecutor for Debugger {
         Box::new(stream) as Box<Stream<Item = (Vec<u8>, MvccInfo), Error = String>>
     }
 
+    fn raw_scan_impl(&self, from_key: &[u8], end_key: &[u8], limit: usize, cf: &str) {
+        let res = self.raw_scan(from_key, end_key, limit, cf).unwrap();
```

Comment:
This will give a fairly unfriendly error message. Do you think adding an `.expect()` and giving it an error reason might be more clear?